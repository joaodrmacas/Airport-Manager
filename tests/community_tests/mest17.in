a LIS Portugal Lisbon
a OPO Portugal Porto
a FAO Portugal Faro
v TP123 LIS OPO 23-03-2022 11:00 00:45 120
v TP555 FAO OPO 24-03-2022 10:00 01:30 105
r TP333 23-03-2022 12345ABCDEF9876 4
q
