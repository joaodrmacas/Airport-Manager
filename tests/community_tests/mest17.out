airport LIS
airport OPO
airport FAO
TP333: flight does not exist
